src/lib/helpers/jsToAqua.ts

```diff
@@ -241,13 +258,13 @@ export const jsToAquaImpl = (
 
       return {
         type: `?${type}`,
-        value: IS_NIL in v && v[IS_NIL] === true ? NULL.value : `?[${value}]`,
+        value: IS_NIL in v && v[IS_NIL] === true ? NIL.value : `?[${value}]`,
         typeDefs,
       };
     }
 
-    const newNestingLevel = nestingLevel === 0 ? 2 : nestingLevel + 1;
-    const ind = INDENTATION.repeat(newNestingLevel);
+    const newNestingLevel = nestingLevel + 1;
+    const newIndent = INDENTATION.repeat(newNestingLevel);
```

Comment:
you already had that defined a few lines above, right?
![Uploading image.png…]()